docs/resources/lakehouse_monitor.md

```diff
@@ -0,0 +1,130 @@
+* `assets_dir` - (Required) - The directory to store the monitoring assets (Eg. Dashboard and Metric Tables)
+* `output_schema_name` - (Required) - Schema where output metric tables are created
+* `baseline_table_name` - Name of the baseline table from which drift metrics are computed from.Columns in the monitored table should also be present in the baseline
+table.
+* `custom_metrics` - Custom metrics to compute on the monitored table. These can be aggregate metrics, derived metrics (from already computed aggregate metrics), or drift metrics (comparing metrics across time windows).
+    * `definition` - [create metric definition]: https://docs.databricks.com/en/lakehouse-monitoring/custom-metrics.html#create-definition
+    * `input_columns` - Columns on the monitored table to apply the custom metrics to.
+    * `name` - Name of the custom metric.
+    * `output_data_type` - The output type of the custom metric.
+    * `type` - The type of the custom metric.
+* `data_classification_config` - The data classification config for the monitor
+* `inference_log` - Configuration for the inference log monitor
+    * `granularities` -  List of granularities to use when aggregating data into time windows based on their timestamp.
+    * `label_col` - Column of the model label
+    * `model_id_col` - Column of the model id or version
+    * `prediction_col` - Column of the model prediction
+    * `prediction_proba_col` - Column of the model prediction probabilities
+    * `problem_type` - Problem type the model aims to solve. Either `PROBLEM_TYPE_CLASSIFICATION` or `PROBLEM_TYPE_REGRESSION`
+    * `timestamp_col` - Column of the timestamp of predictions
+* `snapshot` - Configuration for monitoring snapshot tables.
+* `time_series` - Configuration for monitoring timeseries tables.
+    * `granularities` -  List of granularities to use when aggregating data into time windows based on their timestamp.
+    * `timestamp_col` - Column of the timestamp of predictions
+* `notifications` - The notification settings for the monitor
+* `schedule` - The schedule for automatically updating and refreshing metric tables.
+* `skip_builtin_dashboard` - Whether to skip creating a default dashboard summarizing data quality metrics.
+* `slicing_exprs` - List of column expressions to slice data with for targeted analysis. The data is grouped by each expression independently, resulting in a separate slice for each predicate and its complements. For high-cardinality columns, only the top 100 unique values by frequency will generate slices.
+* `warehouse_id` - Optional argument to specify the warehouse for dashboard creation. If not specified, the first running warehouse will be used.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` -  ID of this monitor is the same as the full table name of the format `{catalog}.{schema_name}.{table_name}`
```

Comment:
We should document the computed fields: `monitor_version`, `drift_metrics_table_name`, `profile_metrics_table_name`, `status`, `dashboard_id`